README.md

```diff
@@ -82,6 +82,85 @@ provided as repository secrets.
           password: ${{ secrets.QUAY_PASSWORD }}
 ```
 
+### Java setup
```

Comment:
I would put everything in a new section of 2nd level (you can replace the `## Travis` section that is currently empty) since this is not specific to migrating from travis but it's good as a starter itself

the section can be called `GitHub Actions starters` (suggestions welcome)